playwright-e2e/tests/dsm/medical-record-request.spec.ts

```diff
@@ -29,8 +34,8 @@ test.describe.serial('Medical records request workflow', () => {
     'Boston Children\'s Hospital'
   ];
 
-  // One CMI Clinical and One CMI Research studies
-  const studies: StudyEnum[] = [StudyEnum.OSTEO2, StudyEnum.PANCAN];
+  // One Clinical studies
+  const studies: StudyEnum[] = [StudyEnum.OSTEO2, StudyEnum.LMS];
```

Comment:
Pancan did not have any participant from Medical Records Not Request Yet quick filter.